apps/emqx_connector/src/schema/emqx_connector_schema.erl

```diff
@@ -56,7 +56,7 @@ enterprise_fields_connectors() -> [].
 
 -endif.
 
-connector_type_to_bridge_types(kafka_producer) -> [kafka_producer];
+connector_type_to_bridge_types(kafka_producer) -> [kafka_producer, kafka];
```

Comment:
@kjellwinblad, it worked fine (I've imported a  config with `bridges.kafka` config, but I haven't verified if listing aliases here follows the initial concept

Reply:
kafka_producer should not be needed in this list since what we are interested in here is Bridge V1 bridges and there is no Bridge V1 bridge type called kafka_producer. Can you remove kafka_producer from this list and write a comment that explains that this function is a mapping from connector types to the corresponding Bridge V1 types?

Reply:
Fixed

Reply:
@kjellwinblad, apparently it broke `emqx_bridge:create/3`, see: https://github.com/emqx/emqx/actions/runs/6695364058/job/18191346775?pr=11842
I've rolled back to `[kafka_producer, kafka]`.